### PR TITLE
Hbase metrics

### PR DESCRIPTION
**Pull request by lavindev**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -266,7 +267,7 @@
         <clojure-data-codec.version>0.1.0</clojure-data-codec.version>
         <clojure-contrib.version>1.2.0</clojure-contrib.version>
         <hive.version>0.14.0</hive.version>
-        <hadoop.version>2.6.1</hadoop.version>
+        <hadoop.version>2.7.3.9.1704132102</hadoop.version>
```
Comment (revans2) [inline pom.xml]: I think this should just be 2.7.3

File: pom.xml [hunk 2]
```diff
@@ -989,6 +991,16 @@
...
+        <repository>
+            <id>central-ymaven</id>
+                <url>http://ymaven.corp.yahoo.com:9999/proximity/repository/public</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+        </repository>
```
Comment (revans2) [inline pom.xml]: This needs to be removed

File: storm-core/pom.xml [hunk 3]
```diff
@@ -33,6 +33,8 @@
         <worker-launcher.conf.dir>/etc/storm</worker-launcher.conf.dir>
         <worker-launcher.additional_cflags></worker-launcher.additional_cflags>
         <argLine></argLine>
+        <hbase.version>0.98.7.27.1704072016_h2</hbase.version>
```
Comment (revans2) [inline storm-core/pom.xml]: This needs to be a public release of hbase

File: storm-core/src/jvm/org/apache/storm/metrics2/store/ConfigKeywords.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package org.apache.storm.metrics2.store;
```
Comment (revans2) [inline storm-core/src/jvm/org/apache/storm/metrics2/store/ConfigKeywords.java]: This needs an apache license header in it

Comment (revans2) [reply storm-core/src/jvm/org/apache/storm/metrics2/store/ConfigKeywords.java]: All of the files really do

**Review by revans2 — COMMENTED** [review]
I did a quick first pass through the code.  These are things that need to change before we can release the code.

We probably also want to squash the commits.

File: storm-core/src/jvm/org/apache/storm/metrics2/store/Metric.java [hunk 5]
```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  * http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```
Comment (abellina) [inline storm-core/src/jvm/org/apache/storm/metrics2/store/Metric.java]: extra `<p>` tag

**Comment by abellina** [comment]
👍

**Pull request merged**